ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -263,6 +264,27 @@ ColumnLayout {
         }
     }
 
+    Component {
+        id: unfurledProfileLinkComponent
+        UserProfileCard {
+            id: unfurledProfileLink
+            readonly property var contact: Utils.parseContactUrl(parent.link)
+            readonly property var contactDetails: Utils.getContactDetailsAsJson(contact.publicKey)
+
+            leftTail: !root.isCurrentUser
+            userName: contact && contact.displayName ? contact.displayName : 
```

Comment:
This should actually use the preferred name, ie. `ProfileUtils.displayName(...)`

Reply:
fixed. Thanks!